mmdet/apis/train.py

```diff
@@ -200,7 +200,9 @@ def _non_dist_train(model,
             seed=cfg.seed) for ds in dataset
     ]
     # put model on gpus
-    model = MMDataParallel(model, device_ids=range(cfg.gpus)).cuda()
+    gpu_ids = range(cfg.gpus) if cfg.gpu_ids is None else cfg.gpu_ids
+    assert len(gpu_ids) == cfg.gpus
```

Comment:
We may deprecate `gpus` if `gpu_ids` is specified.

Reply:
I have made changes to deprecate gpus